src/test/java/marquez/service/models/Generator.java

```diff
@@ -166,14 +168,20 @@ public static DataSourceRow genDataSourceRow() {
         .build();
   }
 
+  // DatasetUrn
+  public static DatasetUrn genDatasetUrn() {
+    return DatasetUrn.from(
+        NamespaceName.fromString("ns" + randNum()), DatasetName.fromString("dataset" + randNum()));
```

Comment:
There is a non-zero chance of a collision. In the past, I've just used a timestamp instead of a random.
Also, maybe it would be a good idea to have the same random value be used for the NS and the DS, b/c it might make correlation between the two easier when debugging.